api/src/main/java/edu/cornell/mannlib/vitro/webapp/utils/dataGetter/SearchFilterValuesDataGetter.java

```diff
@@ -0,0 +1,164 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+package edu.cornell.mannlib.vitro.webapp.utils.dataGetter;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.ServletContext;
+
+import com.google.common.base.Optional;
+import edu.cornell.mannlib.vitro.webapp.config.ConfigurationProperties;
+import edu.cornell.mannlib.vitro.webapp.controller.VitroRequest;
+import edu.cornell.mannlib.vitro.webapp.dao.DisplayVocabulary;
+import edu.cornell.mannlib.vitro.webapp.search.controller.FilterValue;
+import edu.cornell.mannlib.vitro.webapp.search.controller.PagedSearchController;
+import edu.cornell.mannlib.vitro.webapp.search.controller.SearchFilter;
+import edu.cornell.mannlib.vitro.webapp.search.controller.SearchFiltering;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.QueryFactory;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.query.QuerySolutionMap;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.shared.Lock;
+import org.apache.tika.utils.StringUtils;
+
+public class SearchFilterValuesDataGetter extends DataGetterBase implements DataGetter {
+    private static final String defaultTemplate = "browseSearchFilterValues.ftl";
+    private static final String searchFilterUri = "<" + DisplayVocabulary.SEARCH_FILTER_VALUE + ">";
+    private static final Log log = LogFactory.getLog(SearchFilterValuesDataGetter.class);
+
+    String dataGetterURI;
+    String searchFilter;
+    VitroRequest vreq;
+    ServletContext context;
```

Comment:
I know this is copied from other DataGetters, but I am wondering why without access modifier (private)? Is this really planned to be used by other classes from this package? I don't think so

Reply:
Fixed